universal-application-tool-0.0.1/app/assets/javascripts/admin_validation.ts

```diff
@@ -0,0 +1,74 @@
+/** The validation controller provides basic client-side validation of admin form fields. */
+class AdminValidationController {
+  static readonly MULTI_OPTION_QUESTION_FIELD_NAME_CREATE = '#question-settings input[name="newOptions[]"]';
+  static readonly MULTI_OPTION_QUESTION_FIELD_NAME_EDIT = '#question-settings input[name="options[]"]';
+
+  isMultiOptionCreateValid = true;
+  isMultiOptionEditValid = true;

```

Comment:
question (blocking): should these be private?

Reply:
Yes, nice catch. Fixed.